user_guide_src/source/dbmgmt/migration.rst

```diff
@@ -209,7 +209,7 @@ creates is the Pascal case version of the filename.
 
   php spark make:migration <class> [options]
 
-You can use (``make:migration``) with the following options:
+You can use (make:migration) with the following options:
```

Comment:
We use <code>``</code> for source code or CLI commands, so I think the following is better.
I don't know why there are `()` in here.

```suggestion
You can use ``make:migration`` with the following options:
```